src/components/PersonLinks/PersonLinks.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Person } from '../../types';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+export const PersonLinks: React.FC<Props> = ({ person, people }) => {
+  const { slug: id } = useParams();
+
+  const findPersonByName = (name: string | null) => {
+    if (!name) {
+      return;
+    }
+
+    return people.find((item: Person) => item.name === name);
```

Comment:
The `findPersonByName` function searches for a person by name. Ensure that the `name` property is unique within the `people` array to avoid incorrect matches. If names are not unique, consider using a different identifier.